src/OpenApi/build/Microsoft.AspNetCore.OpenApi.targets

```diff
@@ -8,9 +8,11 @@
     <ItemGroup>
       <AdditionalFiles
         Include="@(ReferencePath->'%(RootDir)%(Directory)%(Filename).xml')"
-        Condition="'%(ReferencePath.Extension)' == '.dll' AND
-        Exists('%(ReferencePath.RootDir)%(ReferencePath.Directory)%(ReferencePath.Filename).xml') AND
-        ('%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference')"
+        Condition="
```

Comment:
Can we add a comment here for what we're doing?

Reply:
Funny story I have to revert this because of https://github.com/dotnet/aspnetcore/issues/60783 but I will be sure to add a comment once I've recolved that issue.